src/web/components/CookieBanner.test.tsx

```diff
@@ -15,43 +15,47 @@ jest.mock('@frontend/web/browser/cookie', () => ({
 }));
 
 describe('CookieBanner', () => {
-    const consentCookie = 'GU_TK';
+    // const consentCookie = 'GU_TK';
 
     beforeEach(() => {
         addCookie.mockReset();
         getCookie.mockReset();
     });
 
     afterEach(() => {
-        expect(getCookie).toHaveBeenCalledWith(consentCookie);
+        // expect(getCookie).toHaveBeenCalledWith(consentCookie);
     });
 
-    it('It should render null if consentCookie set', () => {
-        getCookie.mockImplementation(() => true);
-
+    it('It should render null', () => {
```

Comment:
This would be a good use case for `it.skip`

Reply:
I didn't know about `it.skip`. Way cleaner now. Thanks!